LICENSE.Apache2

```diff
@@ -1,201 +1,339 @@
-Apache License
-                           Version 2.0, January 2004
```

Comment:
I think you need to reclone your tree, because we only want that bit at the very bottom and not this license stuff.

Reply:
OK，you can just  manually edit the patch and update the EROFS part at the bottom. i think there is some errors in my git

Reply:
Hey, I fixed up the patch and tested it here. 

One question on the signed off though. The "From" line on the patch has:

From: xinxin shu <shuxinxin@chinac.com>

but the signed off is:

Signed-off-by: Zhaohui Yang <280965257@qq.com>

Are these the same people but just using 2 email addresses and names?

Reply:
the same people ： Zhaohui Yang 280965257@qq.com

Reply:
Thanks merged in:

https://github.com/open-iscsi/tcmu-runner/commit/fc8b065ab79d32d0494384b228abf9a4fe392206